stan/math/rev/fun/mdivide_left_spd.hpp

```diff
@@ -238,6 +238,100 @@ mdivide_left_spd(const EigMat1 &A, const EigMat2 &b) {
   return res;
 }
 
+template <typename T1, typename T2,
+	  require_all_matrix_t<T1, T2>* = nullptr,
+          require_any_var_matrix_t<T1, T2>* = nullptr>
+inline auto mdivide_left_spd(const T1& A, const T2& B) {
+  using ret_val_type = plain_type_t<decltype(value_of(A) * value_of(B))>;
+  using ret_type = var_value<ret_val_type>;
+
+  if (A.size() == 0) {
+    return ret_type(ret_val_type(0, B.cols()));
```

Comment:
`ret_type(0, B.cols())` fails. I forgot what we've done previously. I know sometimes we've returned the zero-sized inputs, but in this case `A` might not have exactly the right size.